zuul.d/edpm.yaml

```diff
@@ -11,11 +11,11 @@
 # Virtual Baremetal job with CRC and single compute node.
 - job:
     name: cifmw-crc-podified-edpm-baremetal
-    nodeset: centos-9-crc-4xlarge
+    nodeset: centos-9-crc-6xlarge
     parent: cifmw-base-crc-openstack
     run: ci/playbooks/edpm_baremetal_deployment/run.yml
     vars:
-      crc_parameters: "--memory 32000 --disk-size 120 --cpus 10"
+      crc_parameters: "--memory 48000 --disk-size 120 --cpus 10"
```

Comment:
Do we want to increase the disk size as well and cpus?

Reply:
@rabi do you maybe know if you need a bigger disk space?

Reply:
We don't need to bump the memory of crc to the max, just increase cpu (12) and disk here... increase the CPU(6) and RAM(8) for EDPM node

Reply:
@rabi updated as per your comment.